src/utils/date.js

```diff
@@ -11,4 +11,8 @@ export const localErrorMessages = {
 
 export const locale = i18n.language;
 
+/* ISO 8601 date format compatible with django filter */
+export const ISODateFormat = date =>
+  `${date.getFullYear() - 1}-${`${date.getMonth() + 1}`.padStart(2, '0')}-${date.getDate()}`;
```

Comment:
Maybe it's shorter in the following way
```js
export const ISODateFormat = date => date.toISOString().split('T')[0]
```

Reply:
I don't like tinkering with strings but I don't have any good argument to oppose, so I did it.